src/views/announcement/_entry.slang

```diff
@@ -25,6 +25,10 @@ article.hentry
         a href="/announcements/#{ announcement.id }" title="#{announcement.created_at.try &.to_s "%b %d, %Y"}"
           = time_ago_in_words(announcement.created_at.not_nil!)
 
+    a href="https://twitter.com/share?text=#{announcement.title}&url=#{SITE.url}#{announcement.short_path}&via=crystallang_ann&related=crystallang_ann" target="_blank"
+      i.fa.fa-retweet
+      = "Retweet"
+
```

Comment:
Let's move this to the list of actions (`span.actions` below) and remove "Retweet". So it will look like this:

<img width="747" alt="screen shot 2017-10-02 at 9 31 50 am" src="https://user-images.githubusercontent.com/3624712/31066313-9c454dbe-a754-11e7-8274-ff3f59d85fdc.png">

Also please add a title to the link (i.e. `a title="Retweet" ...`).

And do not forget to add yourself to the list of [contributors](https://github.com/crystal-community/crystal-ann/#contributors) :)

Reply:
Done :)
Now, all right?